app/src/main/java/by/academy/jc/klimakhovich_ht3/FrogJumper.java

```diff
@@ -0,0 +1,21 @@
+package by.academy.jc.klimakhovich_ht3;
+
+public class FrogJumper {
+    public int frog1(int x, int y, int d) {
+        int step = 0;
+        int i = x;
+        while (i < y) {
+            i += d;
+            step++;
+        }
+        return step;
+    }
+
+    public int frog2(int x, int y, int d) {
```

Comment:
Please provide single best possible solution for the task, or consider to automatically choose what fits better.